templates/server-service.yaml

```diff
@@ -37,7 +37,7 @@ spec:
       {{- if and (.Values.server.service.nodePort) (eq (.Values.server.service.type | toString) "NodePort") }}
       nodePort: {{ .Values.server.service.nodePort }}
       {{- end }}
-    - name: internal
+    - name: {{ include "vault.scheme" . }}-internal
```

Comment:
Was this change necessary for istio? Or just going for consistency?

Reply:
i wasnt utilizing this port, so i am not sure.
however, based on this:
https://istio.io/docs/ops/configuration/traffic-management/protocol-selection/

it says that prefixing port names with "http" or "https" informs istio of the protocol of the traffic. but further down it says that istio can do automatic detection. for the most part this is consistency